src/s2wasm.h

```diff
@@ -58,7 +58,6 @@ class S2WasmBuilder {
         startFunction(startFunction),
         globalBase(globalBase),
         nextStatic(globalBase),
-        minInitialMemory(0),
```

Comment:
`minInitialMemory` is now completely gone. In the previous commit it was always updated exactly when `nextStatic` was updated and exactly to its value. It only had one other use (at the top of `fix()` which now uses `nextStatic` instead. The consequence of unconditionally allocating the stack pointer is that there will never be a case with no globals.